sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -181,7 +182,7 @@ private[hive] class HiveClientImpl(
       logDebug(
         s"""
            |Applying Hadoop/Hive/Spark and extra properties to Hive Conf:
-           |$k=${if (k.toLowerCase(Locale.ROOT).contains("password")) "xxx" else v}
+           |$k=${SQLConf.get.redactOptions(k.toLowerCase(Locale.ROOT) -> v)._2}
```

Comment:
Can't we just use Map here instead of adding the copies of `redact` for (K, V)?

Reply:
> Can't we just use Map here instead of adding the copies of `redact` for (K, V)?

In that way we need to get conf map , put all of then to hiveConf then call `SQLConf.get.redactOptions`, and print out log.

If we don't want to overloading a new method,  it's ok.

Reply:
Yeah I don't think it's worth adding two more methods for this.

Reply:
Change back use origin method.